File: sanidad-backend/src/main/java/com/farmacia/sanidadbackend/dto/UbicacionLoteResponse.java
package com.farmacia.sanidadbackend.dto;

import lombok.Data;

@Data
public class UbicacionLoteResponse {
    private Long id;
    private Long rackId;
    private String rackNombre;
    private Long loteDetalleId;
    private String medicamentoNombre;
    private Integer nivel;
    private Integer columna;
    private Integer profundidadIndex;
    private Integer cantidad;
    private Boolean activo;
}

File: SanidadMovil/app/src/main/java/com/sanidad/movil/data/remote/dto/UbicacionLoteResponse.kt
package com.sanidad.movil.data.remote.dto

data class UbicacionLoteResponse(
    val id: Long,
    val rackId: Long,
    val rackNombre: String,
    val loteDetalleId: Long,
    val medicamentoNombre: String,
    val nivel: Int,
    val columna: Int,
    val profundidadIndex: Int,
    val cantidad: Int,
    val activo: Boolean
)